### PR TITLE
Adding -i INCLUDEFILES argument

### PR DESCRIPTION
**Pull request by SebastianLopienski**

### CODE REVIEW

File: courseradownloader/courseradownloader.py [hunk 1]
```diff
@@ -79,10 +81,12 @@ def __init__(self,username,
         self.password = password
         self.parser = parser
 
-        # Split "ignorefiles" argument on commas, strip, remove prefixing dot
+        # Split "ignorefiles" and "includefiles" arguments on commas, strip, remove prefixing dot
         # if there is one, and filter out empty tokens.
         self.ignorefiles =  [x.strip()[1:] if x[0]=='.' else x.strip()
                              for x in ignorefiles.split(',') if len(x)]
+        self.includefiles = [x.strip()[1:] if x[0]=='.' else x.strip()
```
Comment (dgorissen) [inline courseradownloader/courseradownloader.py]: I would avoid the copy-paste here

Comment (SebastianLopienski) [inline courseradownloader/courseradownloader.py]: How? With a separate function to split, filter, strip etc.? Would it be a gain (in terms of code readability/maintainability)?

Comment (dgorissen) [inline courseradownloader/courseradownloader.py]: I agree its debatable but I would opt for a separate function.

Comment (SebastianLopienski) [inline courseradownloader/courseradownloader.py]: OK, implemented, indeed looks cleaner.

**Pull request merged**